components/gallery/GalleryItemAction/GalleryItemActionType/GalleryItemOffer.vue

```diff
@@ -103,7 +103,7 @@ const price = ref('')
 const offerPrice = ref()
 const active = ref(false)
 const confirm = ref(false)
-const days = [1, 3, 7, 14, 30]
+const days = [7, 14, 30]
```

Comment:
we're removing some days on all screen?

Reply:
yes, according to the description in the original issue.